### DISCUSSION TITLE
screen resolution

### DISCUSSION THREAD

**Discussion by zhdali** [zbm-dev/zfsbootmenu#210] [Questions] [answered]
Hi.
I'm testing zbm on my laptop, which has a 3k screen, the menu is practically impossible to read. 
I am using the EFI version of zbm, starting it from rEFIND, is there a way to specify the resolution to use in "refind_linux.conf"?
For example with the artix live i can add "video = 1024x576" in the kernel parameters.
Can something like this be done with ZBM too?

**Comment by ahesford** [comment, marked as answer]
ZBM is just a self-contained Linux environment, so anything you do to make your terminal legible on your regular system can be adapted.

I work around this problem with fonts. My ZBM image has the Terminus console font installed (this happens by default on my system) and I set `rd.vconsole.font=ter-132b` on the kernel command line.

**Reply by zhdali** [reply]
Thanks, your suggestion works perfectly.

**Comment by zdykstra** [comment]
Are you using the binary release, or did you build your own copy? The license for Terminus looks permissive enough to allow us to bundle it in the upcoming release.
